pkgs/build-support/build-dotnet-package/default.nix

```diff
@@ -98,7 +101,10 @@ attrsOrig @
             [ -f "$exe" ] || continue
             mkdir -p "$out"/bin
             commandName="$(basename -s .exe "$(echo "$exe" | tr "[A-Z]" "[a-z]")")"
-            makeWrapper "${mono}/bin/mono \"$exe\"" "$out"/bin/"$commandName"
+            makeWrapper \
+              "${mono}/bin/mono \"$exe\"" \
+              "$out"/bin/"$commandName" \
+              ''${makeWrapperArgs}
```

Comment:
What do you mean? If you're talking about `${makeWrapperArgs}` it's environment variable expanded by bash.